nexus/tests/integration_tests/instances.rs

```diff
@@ -6891,7 +6891,7 @@ pub async fn instance_wait_for_state_as(
     instance_id: InstanceUuid,
     state: omicron_common::api::external::InstanceState,
 ) -> Instance {
-    const MAX_WAIT: Duration = Duration::from_secs(120);
+    const MAX_WAIT: Duration = Duration::from_secs(320); // 120 was not enough
```

Comment:
again I think the comment isn't necessary here

Reply:
Removed